base/operators.jl

```diff
@@ -68,12 +68,16 @@ Supertype operator, equivalent to `T2 <: T1`.
 """
     supertype(T::DataType)
```

Comment:
Keep this consistent with the text below. So it should be, e.g., `supertype(T::Union{DataType,UnionAll})`, I guess.

Reply:
I think usually when there's more than one method there's no type annotation? No idea ^^

Reply:
IMO it would maybe be best to put `supertype(T::Type)` here, but clarify in the description below that only UnionAll and DataType are accepted. You could also explicitly tell that `Union`s aren't allowed.